recipes-multimedia/python3-picamera2/python3-picamera2_git.bb

```diff
@@ -0,0 +1,22 @@
+SUMMARY = "Python interface to the Raspberry Pi camera module via libcamera"
+DESCRIPTION = "This package provides a pure Python interface to the Raspberry Pi camera module via libcamera for Python 3"
+HOMEPAGE = "https://github.com/raspberrypi/picamera2" 
+
+LICENSE = "BSD-2-Clause"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=6541a38108b5accb25bd55a14e76086d"
+
+RDEPENDS:${PN} = "python3-numbers   \
+                  python3-ctypes    \
+                  python3-colorzero \
+                  picamera-libs     \
+"
+
+SRC_URI = "git://git@github.com/raspberrypi/picamera2.git;protocol=ssh;branch=main"
```

Comment:
does it have to use ssh protocol for SRC_URI, can we use https instead? something like
`git://github.com/raspberrypi/picamera2.gitprotocol=https;branch=main`